packages/ui-components/src/AddressSummary/AddressSummary.tsx

```diff
@@ -52,23 +52,23 @@ export class AddressSummary extends React.PureComponent<Props> {
 
     if (orientation === 'vertical') {
       return (
-          <Stacked>
-            {
-              <IdentityIcon value={address as string || PLACEHOLDER_ADDRESS} theme={'substrate'} size={styles.identiconSize} />
-            }
-                <Name fontSize={styles.nameSize}> { name || PLACEHOLDER_NAME } </Name>
-                <BalanceDisplay balance={balance} />
-          </Stacked>
+        <Stacked>
+          {
```

Comment:
Fly-by comment... :)

And more interested than anything else, why -

```
{
  <IdentityIcon ... />
}
```

as opposed to

```
<IdentityIcon ... />
```